class-components/src/components/ErrorBoundary/tests/ErrorBoundary.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
Please don't place tests in a separate `tests` folder. It's okay for a folder, e.g. ErrorBoundary to contain these files:
ErrorBoundary.tsx
ErrorBoundary.type.ts
ErrorBoundary.test.tsx